Header header
int64 left_ticks
int64 right_ticks
int64 left_speed_ticks_per_s
int64 right_speed_ticks_per_s
